Gemfile.lock

```diff
@@ -311,69 +312,73 @@ GEM
       actionpack (>= 5.0)
       railties (>= 5.0)
     rexml (3.2.5)
-    rubocop (1.26.1)
+    rubocop (1.35.1)
+      json (~> 2.3)
       parallel (~> 1.10)
-      parser (>= 3.1.0.0)
+      parser (>= 3.1.2.1)
       rainbow (>= 2.2.2, < 4.0)
       regexp_parser (>= 1.8, < 3.0)
-      rexml
-      rubocop-ast (>= 1.16.0, < 2.0)
+      rexml (>= 3.2.5, < 4.0)
+      rubocop-ast (>= 1.20.1, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 3.0)
-    rubocop-ast (1.16.0)
+    rubocop-ast (1.21.0)
       parser (>= 3.1.1.0)
-    rubocop-performance (1.13.3)
+    rubocop-performance (1.14.3)
       rubocop (>= 1.7.0, < 2.0)
       rubocop-ast (>= 0.4.0)
     ruby-progressbar (1.11.0)
     rubyzip (2.3.2)
-    selenium-webdriver (4.1.0)
+    selenium-webdriver (4.4.0)
       childprocess (>= 0.5, < 5.0)
       rexml (~> 3.2, >= 3.2.5)
-      rubyzip (>= 1.2.2)
-    sidekiq (6.4.1)
-      connection_pool (>= 2.2.2)
+      rubyzip (>= 1.2.2, < 3.0)
+      websocket (~> 1.0)
+    sidekiq (6.5.6)
+      connection_pool (>= 2.2.5)
       rack (~> 2.0)
-      redis (>= 4.2.0)
+      redis (>= 4.5.0, < 5)
     simple_form (5.1.0)
       actionpack (>= 5.2)
       activemodel (>= 5.2)
-    sshkit (1.21.2)
+    sshkit (1.21.3)
```

Comment:
### [sshkit](http://github.com/capistrano/sshkit)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.21.2 → 1.21.3

[_[change-log](https://github.com/capistrano/sshkit/releases), ~~source-code~~, [gem-diff](https://my.diffend.io/gems/sshkit/1.21.2/1.21.3)_]





<details>
<summary>Commits</summary>

A change of **8** commits. See the full changes on [the compare page](https://github.com/capistrano/sshkit/compare/v1.21.2...v1.21.3).

These are the individual commits:
- (f752880) [Upgrade to GitHub-native Dependabot](https://github.com/capistrano/sshkit/commit/f7528802e655890b1f4b25a83e54b1153281a04b)
- (7473458) [Merge pull request #492 from capistrano/dependabot/add-v2-co…](https://github.com/capistrano/sshkit/commit/7473458b0422e27dbcadcdcfcffa65fa46bb64c1)
- (c59db4e) [Show how to change the SSH port in the DSL](https://github.com/capistrano/sshkit/commit/c59db4e05ce50246692123cee7740f1a40093c14)
- (74cc92b) [Merge pull request #494 from bretweinraub/master](https://github.com/capistrano/sshkit/commit/74cc92b2e7f987ba02853959e33bbdfba07ee9e2)
- (fd05255) [chore: Included githubactions in the dependabot config (#497…](https://github.com/capistrano/sshkit/commit/fd0525576fe15aa2b6fe07f0bddab8bbae8f2e54)
- (93c70d8) [Bump toolmantim/release-drafter from 5.2.0 to 5.20.0 (#498)](https://github.com/capistrano/sshkit/commit/93c70d882b4dba3a6bee987cf0e83c4e04f9d25a)
- (7f2ff82) [Bump toolmantim/release-drafter from 5.20.0 to 5.20.1 (#500)](https://github.com/capistrano/sshkit/commit/7f2ff82706f64475a887d8f4eacbb98bcd01c74c)
- (2597aa7) [Release 1.21.3](https://github.com/capistrano/sshkit/commit/2597aa7c8c2fc7344a70bee36bd2d9242d91b3bb)


</details>